app/components/Layout/Flex.tsx

```diff
@@ -1,6 +1,6 @@
 import cx from 'classnames';
 import styles from './Flex.css';
-import type { ReactNode, ElementType } from 'react';
+import type { ReactNode, ElementType, HTMLAttributes } from 'react';
```

Comment:
I now realized that I'd prefer if our import sort put `react` imports at the top and style imports at the bottom. Just a thought.

Reply:
I now also realized this is a type import, so forget what I said about react, but the style import remains.